external-crates/move/crates/move-package-alt/tests/snapshots/manifest_parsing_test__Move.toml.snap

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
I don't see the test files, I bet we need to add `!Move.toml` and `!Move.lock` to the gitignore for the tests